src/components/App.js

```diff
@@ -79,6 +88,15 @@ export class App extends Component {
       case 'producer':
       case 'educator':
       case 'admin':
+        // unless the user is a viewer
+        if (userType === USER_TYPE_VIEWER) {
+          return (
+            <>
+              {headerVisible || standalone ? <Header /> : null}
```

Comment:
It might become interesting to put this header in `StudentMode` directly or in `StudentView`. Unless you want to do something different with the header.

Reply:
I don't remember why we did it like this... but you're right. I've gone ahead and changed this.